### PR TITLE
Jk random

### PR DESCRIPTION
**Pull request by josephkane**
Random player and enemy generators working, added enemy generator to fight button

### CODE REVIEW

File: javascripts/characters.js [hunk 1]
```diff
@@ -32,5 +33,27 @@ var Gauntlet = (function(gauntlet) {
     return character;
   };
 
+  gauntlet.Character.randomCharacter = function(name) {
+    let randomName = getRandElem(name);
+    let randomSpecies = getRandElem(gauntlet.Species.getSpeciesList());
+    let allowedClasses = Gauntlet.Species.getSpeciesList().allowedClasses;
```
Comment (aoswalt) [inline javascripts/characters.js]: allowedClasses won't work here. The property is on an individual species, not a list.

File: javascripts/characters.js [hunk 2]
```diff
@@ -32,5 +33,27 @@ var Gauntlet = (function(gauntlet) {
...
+    if(!allowedClasses) {
+      allowedClasses = Gauntlet.PlayerClass.getClassList();
+    };
+    let randomClass = getRandElem(allowedClasses);
+    let allowedWeapons = Gauntlet.PlayerClass.getClassList().allowedWeapons;
```
Comment (aoswalt) [inline javascripts/characters.js]: Same here with allowedWeapons

Comment (josephkane) [inline javascripts/characters.js]: @aoswalt Ah, I totally see that and why. I think I fixed it. Thanks!

File: javascripts/characters.js [hunk 3]
```diff
@@ -14,9 +14,10 @@ var Gauntlet = (function(gauntlet) {
   gauntlet.Character.prototype.attack = function(enemy) {
     let weapDamage = this.weapon.swing()
     enemy.health -= weapDamage;
-    console.log("Attack: ", `${this.name} attacks ${enemy.name} with ${this.weapon.name} for ${weapDamage} damage.`);
+    // console.log("Attack: ", `${this.name} attacks ${enemy.name} with ${this.weapon.name} for ${weapDamage} damage.`);
+    $("")
```
Comment (aoswalt) [inline javascripts/characters.js]: Typo?

Comment (josephkane) [inline javascripts/characters.js]: @aoswalt SON-OF-A-BITCH

**Comment by aoswalt** [comment]
:+1:    Aside from the typo... :-1:

**Pull request merged**